milter/client/meson.build

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2022  Daijiro Fukuda <fukuda@clear-code.com>
+# Copyright (C) 2022  Sutou Kouhei <kou@clear-code.com>
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+sources = files(
+  'milter-client-context.c',
+  'milter-client-main.c',
+  'milter-client-runner.c',
+  'milter-client-single-thread-runner.c',
+  'milter-client.c',
+)
+
+headers = files(
+  'milter-client-context.h',
+  'milter-client-objects.h',
+  'milter-client-runner.h',
+  'milter-client-single-thread-runner.h',
+  'milter-client.h',
+)
+
+install_headers(headers, subdir: join_paths('milter', 'client'))
+
+enums = gnome.mkenums_simple('milter-client-enum-types',
+                             body_prefix: '#include <config.h>',
+                             identifier_prefix: 'Milter',
```

Comment:
```suggestion
                             identifier_prefix: 'MilterClient',
```

Reply:
Actually, I thought it too.
However, the binding codes use `MILTER_TYPE_CLIENT_EVENT_LOOP_BACKEND` and so on.

* https://github.com/milter-manager/milter-manager/blob/master/binding/ruby/ext/client/rb-milter-client.c#L378

So we have to use `Milter` for this prefix.
Using `MilterClient` here will generate `MILTER_CLIENT_TYPE_EVENT_LOOP_BACKEND` and so on.

Reply:
Hmm, something is wrong. I will check again.

Reply:
Confirmed.
It appears we need to use just `Milter` here.

Reply:
If using `MilterClient`, then the following error occurs:

```
FAILED: milter/client/2b66477@@milter-client@sha/milter-client-main.c.o 
cc -Imilter/client/2b66477@@milter-client@sha -Imilter/client -I../milter-manager/milter/client -I. -I../milter-manager/ -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -g -fPIC -DEV_COMPAT3=0 -DHAVE_CONFIG_H '-DMILTER_LOG_DOMAIN="milter-client"' -MD -MQ 'milter/client/2b66477@@milter-client@sha/milter-client-main.c.o' -MF 'milter/client/2b66477@@milter-client@sha/milter-client-main.c.o.d' -o 'milter/client/2b66477@@milter-client@sha/milter-client-main.c.o' -c ../milter-manager/milter/client/milter-client-main.c
../milter-manager/milter/client/milter-client-main.c: In function ‘parse_event_loop_backend’:
../milter-manager/milter/client/milter-client-main.c:463:37: error: ‘MILTER_TYPE_CLIENT_EVENT_LOOP_BACKEND’ undeclared (first use in this function); did you mean ‘MILTER_TYPE_EVENT_LOOP_BACKEND’?
  463 |     backend_enum = g_type_class_ref(MILTER_TYPE_CLIENT_EVENT_LOOP_BACKEND);
```

Reply:
Hmm, I used wrong naming style... I should have used `MilterClientClient` or something to use `MilterClient` as prefix... 

OK. We use `Milter` as milter-client's prefix. It generates `Client` class under `MilterClient` namespase through...

Reply:
I see, thanks.